scripts/lb-wrapper

```diff
@@ -2,44 +2,41 @@
 
 # Configuration
 XKLB_EXECUTABLE="${XKLB_EXECUTABLE:-lb}"
-TMP_DOWNLOADS_PATH="/library/downloads/calibre-web"
-LOG_FILE="/var/log/xklb.log"
-SURVEY_DB_FILE="${TMP_DOWNLOADS_PATH}/survey.db"
 URL="$1"
+LOG_FILE="/var/log/xklb.log"
+TMP_DOWNLOADS_DIR="/library/downloads/calibre-web"
+SURVEY_DB_FILE="${TMP_DOWNLOADS_DIR}/survey.db"
 
-mkdir -p ${TMP_DOWNLOADS_PATH}
+mkdir -p ${TMP_DOWNLOADS_DIR}
 
 # Function to log messages
 log() {
     echo "$(date +'%Y-%m-%d %H:%M:%S') - $1" | tee -a ${LOG_FILE}
 }
 
 # Check if xklb is installed
-if ! command -v "${XKLB_EXECUTABLE}" &> /dev/null
-then
+if ! command -v "${XKLB_EXECUTABLE}" &> /dev/null; then
```

Comment:
@ddeldesir I don't think `&> /dev/null;` does much anything in this case?

```suggestion
if ! command -v "${XKLB_EXECUTABLE}"; then
```

(Then again it's also harmless...)

Reply:
Ok, I get it.